src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -57,6 +57,24 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
 
   /** The in-memory list of Users. */
   private List<User> users;
+  
+  /** Access the current amount of users **/
+  public int getNumUsers() {
+	  return users.size();
+  }
+  
+  public String getNewest() {
+		String retValue = "";
+		if (users.size() != 0) {
+			User temp = users.get(0);
```

Comment:
`temp` is usually a variable name to avoid, since it doesn't really tell me anything about the variable. Something like `newestUser` would be better